components/integration/replace-zapier/code/zapier-bbe/better-security.md

```diff
@@ -1,50 +1,52 @@
 ---
 title: 'Better security'
 description: 'Ballerina provides robust security features such as encryption, authentication, and authorization, which are essential for businesses dealing with sensitive data.'
-url: 'https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-zapier-samples/gdrive-new-event-to-slack-message'
+url: 'https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-zapier-samples/github-summary-via-secure-service'
```

Comment:
In other pages, the urls points to the main.bal
eg:- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/components/ai/code/ai-bbe/create-ai-powered-apis.md

Reply:
Fixed, thanks